src/components/pages/home/AddressLink.tsx

```diff
@@ -0,0 +1,52 @@
+import { useContext } from "react";
+import Link from "next/link";
+import { AddressContext } from "@/components/lib/context/AddressContext";
+
+interface AddressLinkProps {
+  type: string;
+  label: string;
+  address: string | null;
+  href: string;
+  children: React.ReactNode;
+}
+
+const AddressLink = ({
+  type,
+  label,
+  address,
+  href,
+  children,
+}: AddressLinkProps) => {
+  const { state, dispatch } = useContext(AddressContext);
+
+  const handleMouseEnter = () => {
+    dispatch({ type: "SET_HOVERED_ADDRESS", payload: address });
+  };
+
+  const handleMouseLeave = () => {
+    dispatch({ type: "SET_HOVERED_ADDRESS", payload: null });
+  };
+
+  const className = `truncate ${
+    type === "l1-l2"
+      ? "max-w-28 md:max-w-48 xl:max-w-28"
+      : type === "latest-transactions" && label === "tx"
+        ? "max-w-60 md:max-w-96 xl:max-w-60"
+        : "max-w-36 md:max-w-48 xl:max-w-36"
+  }`;
+
+  return (
+    <Link
+      href={href}
+      className={`flex items-center text-primary hover:brightness-150 ${
```

Comment:
Can you try replicating the Etherscan style using tailwind classes?
Something like a yellow dashed border and light yellowish background with high opacity.
Shouldn't be too hard and let's stick to an existing style before coming up with our own.

![Screenshot 2024-06-24 at 12 50 10](https://github.com/walnuthq/op-scan/assets/5597359/a69fbc5c-31d2-4406-8f1f-d2d28a24856b)

Reply:
Of course, I'll be working on it. Thanks for the feedback!